### ISSUE TITLE
fix: Linkedin Icon

### ISSUE THREAD

**Issue by RayyanRNKJ** [EddieHubCommunity/LinkFree#675]
### Description

The **Linkedin Icon** is missing in the link section,
It is missing without hovering, while hovering nothings changed.

### Screenshots

![Group 15](https://user-images.githubusercontent.com/64679235/141964274-4ad5d276-adc6-415e-94d0-dfd9cf5ffca1.png)

**Comment by Vyvy-vi** [comment]
duplicate of #130 #554 #106
dependent on #336

**Comment by RayyanRNKJ** [comment]
> duplicate of #130 #554 #106 dependent on #336

Sorry, I haven't searched this issue enough.
My bad.

**Comment by RayyanRNKJ** [comment]
> duplicate of #130 #554 #106 dependent on #336

Sorry, My bad.
I haven't searched this issue enough.
